### PR TITLE
Mods to use pega-auth

### PR DESCRIPTION
**Pull request by vrseraphin**
Web Components SDK should also move to using @pega/auth library

### CODE REVIEW

File: src/bridge/BridgeBase/index.ts
Comment (niallriddell) [inline src/bridge/BridgeBase/index.ts:1]: These is a duplicate of the same change in the playwright PR

File: src/samples/SimplePortal/SimpleMain/index.ts
Comment (niallriddell) [inline src/samples/SimplePortal/SimpleMain/index.ts:1]: is asyncFirstUpdated now handled in pega auth?

File: src/bridge/BridgeBase/index.ts
Comment (vrseraphin) [reply src/bridge/BridgeBase/index.ts:1]: Yes, it is.  This change came first and then you had asked about playwright tests...and found it was needed there as well.

File: src/samples/SimplePortal/SimpleMain/index.ts
Comment (vrseraphin) [reply src/samples/SimplePortal/SimpleMain/index.ts:1]: Not familiar with asyncFirstUpdated.  What is that.....did I change something related to that?

Comment (niallriddell) [reply src/samples/SimplePortal/SimpleMain/index.ts:1]: Yes you commented it out

**Review by niallriddell — APPROVED** [review]

File: src/samples/SimplePortal/SimpleMain/index.ts
Comment (vrseraphin) [reply src/samples/SimplePortal/SimpleMain/index.ts:1]: Code was commented out that is not currently being utilized and was attempting to retrieve the current authorization header (which is no longer supported)....to make direct DX API calls.  Rather expectation is that only in @pega/auth will such invocations occur (if needed) and really should be using PCore to make any/all DX API calls.

**Pull request merged**